.github/workflows/ci-cifuzz.yml

```diff
@@ -0,0 +1,26 @@
+name: CIFuzz
+on: [pull_request]
```

Comment:
Please add the following similar to our other CI jobs, e.g. https://github.com/dgraph-io/dgraph/blob/main/.github/workflows/ci-dgraph-load-tests.yml

```
on:
  push:
    branches:
      - main
      - 'release/**'
  pull_request:
    types:
      - opened
      - reopened
      - synchronize
      - ready_for_review
    branches:
      - main
      - 'release/**'
  schedule:
    - cron: "0 0,6,12,18 * * *"
```